### ISSUE TITLE
Storybook requires installing webpack which makes no sense

### ISSUE THREAD

**Issue by sebastian-nowak** [storybookjs/storybook#10315]
**Describe the bug**
Storybook has a peerDependency on babel-loader which has a peerDependency on webpack => 2. The result is that storybook requires installing webpack as a dependency even though it's absolutely not needed.

```
▶ yarn
➤ YN0000: ┌ Resolution step
➤ YN0002: │ test@workspace:library/test doesn't provide webpack@>=2 requested by babel-loader@npm:8.1.0
➤ YN0000: └ Completed in 10.43s
➤ YN0000: ┌ Fetch step
```

**Expected behavior**
Storybook should not have a (transitive) peerDependency on webpack.

**Additional context**
I'm trying out storybook 6.0.0-alpha.30 with yarn2. Yarn complains about the unsatisfied peerDependency.

**Comment by shilman** [comment]
cc @gaetanmaisse

**Comment by gaetanmaisse** [comment]
Hmmm I see, it appears because I fixed some missing transitive peer dependencies (great post about this: https://dev.to/arcanis/implicit-transitive-peer-dependencies-ed0) and as `@storybook/core` has a peerDependency on `babel-loader` almost every package has it too 🙃 

However, it looks like `babel-loader` is now a regular dependency of `@storybook/core`(https://github.com/storybookjs/storybook/commit/997c224d5528235280ced846bdef4cc148a9514b#diff-d56180255ddcda2cd5bc64ca50a485f1R51) so we can maybe clean its usage 🤔 . I will discuss this with @ndelangen today.

**Comment by sebastian-nowak** [comment]
This is still an issue I believe.

**Comment by gaetanmaisse** [comment]
https://github.com/storybookjs/storybook/pull/10613 has been merged, so as soon `6.0.0-beta.1` will be out this issue should be fixed, as `babel-loader` will not be a peerDeps of `@storybook/core` anymore.
